examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,524 @@
+{
```

Comment:
<p>train_ppl/lr is still  too easy to decipher. Why not <code>dsfglkjgf;l</code>? Make your readers struggle</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>